cmd/loadtest/log.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sync"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/go-kit/kit/log/term"
+)
+
+// MultiLogger logs to multiple log.Logger instances on a single Log() call.
+type MultiLogger struct {
+	log.Logger
+
+	loggers []log.Logger
+}
+
+func NewMultiLogger(loggers ...log.Logger) *MultiLogger {
+	return &MultiLogger{
+		loggers: append([]log.Logger{}, loggers...),
+	}
+}
+
+func (l *MultiLogger) Log(keyvals ...interface{}) error {
+	kvs := keyvals[:len(keyvals):len(keyvals)]
```

Comment:
not sure what this index does: `:len(keyvals):len(keyvals)`

Reply:
> not sure what this index does: :len(keyvals):len(keyvals)

From https://golang.org/ref/spec#Slice_expressions:

> `a[low : high : max]`
constructs a slice of the same type, and with the same length and elements as the simple slice expression a[low : high]. Additionally, it controls the resulting slice's capacity by setting it to `max - low`. Only the first index may be omitted; it defaults to 0. After slicing the array a